### PR TITLE
Add npm run scripts

### PR DESCRIPTION
**Pull request by deiga**
It also adds `gulp-cli` dependency, for without the build caused some issues

### CODE REVIEW

File: src/api_server/secrets.js [hunk 1]
```diff
@@ -30,11 +30,11 @@ const SECRETS_MAPPING = {
      * HTTPS_CERT_FILE = /etc/letsencrypt/live/<YOUR_DOMAIN_NAME>/fullchain.pem
      */
     HTTPS_KEY_FILE: {
-        dev: '',
+        dev: '/Users/timosand/certs/localhost-key.pem',
```
Comment (zowesiouff) [inline src/api_server/secrets.js]: Those strings shouldn't be merged upstream: too specific to your particular setup. There might be value in having them relative to make them "fit" with the setup docs though.

File: src/api_server/secrets.js [hunk 2]
```diff
@@ -30,11 +30,11 @@ const SECRETS_MAPPING = {
...
         prod: '',
     },
     HTTPS_CERT_FILE: {
-        dev: '',
+        dev: '/Users/timosand/certs/localhost-cert.pem',
```
Comment (zowesiouff) [inline src/api_server/secrets.js]: same :)

File: src/api_server/secrets.js [hunk 3]
```diff
@@ -77,11 +77,11 @@ const SECRETS_MAPPING = {
      * https://developers.facebook.com/apps
      */
     FACEBOOK_CLIENT_ID: {
-        dev: '',
+        dev: '1858203781100196',
```
Comment (zowesiouff) [inline src/api_server/secrets.js]: same :)

File: src/api_server/secrets.js [hunk 4]
```diff
@@ -77,11 +77,11 @@ const SECRETS_MAPPING = {
...
         prod: '',
     },
     FACEBOOK_CLIENT_SECRET: {
-        dev: '',
+        dev: 'e1136bfe0ed10aefb11c6a4abdd5ef72',
```
Comment (zowesiouff) [inline src/api_server/secrets.js]: You should never commit secrets like that to a public repo (Now, I highly advise that you rotate that secret through FB's developer portal).

File: src/api_server/secrets.js [hunk 5]
```diff
@@ -104,8 +104,8 @@ const SECRETS_MAPPING = {
      * Google OAuth app secrets (required)
      * https://console.developers.google.com/projectselector/apis/credentials
      */
-    GOOGLE_CLIENT_ID: '',
-    GOOGLE_CLIENT_SECRET: '',
+    GOOGLE_CLIENT_ID: '416349265658-i2b7j6gn60srlnfuah76gpll90s4il62.apps.googleusercontent.com',
+    GOOGLE_CLIENT_SECRET: 'f0woWshFE86s3FaikedqAVAd',
```
Comment (zowesiouff) [inline src/api_server/secrets.js]: same :)

**Comment by zowesiouff** [comment]
All that said, I would argue that typing `npm run build:dev` instead of `NODE_ENV='dev' gulp servers` isn't a massive win? Though maybe there is value in the fact that it's documenting the build inside `package.json` instead of in docs? Or am I missing something valuable about npm scripts?

**Comment by deiga** [comment]
@zowesiouff 
First off, I apologize for adding those configs. I was fairly sure I didn't commit them. Will fix :)

`npm run build:dev` vs `NODE_ENV='dev' gulp servers` has the advantage of not needing `gulp` in the current path (f.ex. not needed to do a global install)

It also documents the command "better" as it's more abstract and enables changing the underlying tools more easily without disrupting development workflow.

Last but not least, it provides a choice to the developer how they want to call the flow :)

**Comment by zowesiouff** [comment]
@deiga
I will merge it for not needing gulp in the current path and the more accessible documentation which are 2 very good points 💯 

That said, it might be worth putting `gulp pm` in the `build:dev` script: it's a build artefact with paths hardcoded in it to other build artefacts (+ it reduces the commands surface: `start:dev` is not useful anymore)

PS: I'm not completely sold on the last 2 points though but they aren't PR-breakers :) 
- changing the underlying tools made easier: considering how volatile the JS ecosystem is, I don't see `package.json` scripts being more abstract or really decoupling anything, even though yarn seems to accept them, the next-hyped-thing&trade; can decide to completely break that contract.
- providing a choice to the developer is great but it increases the maintainability burden (making sure all those different "choices" continue to work). In this specific case, it's not a big issue, Kin OSS is still young and all mentions of the underlying gulp commands can be replaced from existing docs to limit the exposed API surface.

**Comment by deiga** [comment]
@zowesiouff Fixed your comments :)

**Comment by zowesiouff** [comment]
LGTM + I learned that you could run several gulp tasks concurrently via a single invocation so double thanks @deiga ;)

**Pull request merged**